upgrading/topics/keycloak/changes-21_0_0.adoc

```diff
@@ -49,3 +49,10 @@ version of Keycloak,
 work with the metadata generated by older Keycloak versions. This mode can be
 enabled using `-Dkeycloak.saml.deprecated.encryption=true` flag. Note this
 backward compatibility mode is planned to be removed in Keycloak 24.
+
+= Deprecated methods from user session provider were removed
+
+In Keycloak 13 there was introduced UserLoginFailureProvider and some methods from 
```

Comment:
Please use backticks for all class names in this text, e.g. \`UserLoginFailureProvider\`